### ISSUE TITLE
Create a live event landing page

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#70]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To publish a page about opening of a new medical center via page builder.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #69, #61, #62, #63, #65, #66, #67

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Implemented through f802f959b517b47e2cfcbeb2c37d995b65a6c5a7.
